pages/linux/tor.md

```diff
@@ -0,0 +1,35 @@
+# tor
+
+> Tor is free software for enabling anonymous communication.
```

Comment:
I think the description should be changed. See [Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#example-descriptions).
```suggestion
> Enable anonymous communication through the Tor network.
```